releasenotes/notes/fix-another-flask-body-blocking-acd533aa3c1bd46e.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    ASM: fix another body read blocking problem with Flask.
```

```suggestion
    ASM: do not block when reading from an empty ``wsgi_input`` with Flask.
```

not entirely sure if this is great, but I think it provides more context for anyone reading the release notes.

---

Updated the changelog with another message, thanks for your feedback!